Dockerfile-bcoin

```diff
@@ -1,30 +1,59 @@
 FROM nfnty/arch-mini:latest AS base
 
 RUN mkdir -p /code/node_modules/bcoin /data
-WORKDIR /code/node_modules/bcoin
-ENTRYPOINT ["node"]
-CMD ["/code/scripts/docker-bcoin-init.js"]
+WORKDIR /code
 
 RUN pacman -Sy --noconfirm archlinux-keyring && \
-    pacman -Syu --noconfirm nodejs npm && \
+    pacman -Syu --noconfirm nodejs-lts-carbon npm && \
```

Comment:
A bug was introduced into node today - https://github.com/nodejs/node/issues/20921
Decided to switch to the `lts` version so we know it will be stable